cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -35,7 +34,7 @@ func (suite *InstallerIntegrationTestSuite) TestInstallFromLocalSource() {
 	// Run installer with source-path flag (ie. install from this local path)
 	cp := ts.SpawnCmdWithOpts(
 		ts.InstallerExe,
-		e2e.WithArgs(target, "--source-path", suite.payloadDir),
+		e2e.WithArgs(target),
```

Comment:
`target` is pointing at an installer in `workdir/installation`, but the payload you're setting up below uses the dir the InstallerExe is in which is `basedir`. These two do not seem to match. If the test still passes it's likely because it's using the files meant to run the state tool as oppose to the payload you've set up.

Reply:
The dir for `InstallerExe` is a separate directory now at `base/installer`. That's the change from [here](https://github.com/ActiveState/cli/pull/1903/files#diff-3d51f7fa49a5898d7c83fcf6ff24388448fd98b3c41e0925b3c0ac90c9fe84f2L210)